data/journal_check/bug_refs.json

```diff
@@ -0,0 +1,77 @@
+{
+    "bsc#1022527": {
+        "description": ".*wickedd.*ni_process_reap.*blocking waitpid.*",
+        "products": ["sle-micro", "opensuse", "microos", "sle"],
+        "type": "feature"
+    },
+    "bsc#000000": {
+        "description": "health-checker/fail.sh check\" failed|Machine didn't come up correct, do a rollback",
+        "products": ["sle-micro", "microos"],
+        "type": "feature"
+    },
+    "bsc#1126272": {
+        "description": "Failed unmounting \/\\S+\\.|-- Reboot --|pam_systemd.*Failed to release session",
+        "products": ["sle-micro", "microos"],
+        "type": "bug"
+    },
+    "bsc#1022525": {
+        "description": ".*rpcbind.*cannot(.*open file.*rpcbind.xdr.*|.*open file.*portmap.xdr.*|.*save any registration.*)",
+        "products": ["sle-micro"],
+        "type": "bug"
+    },
+    "bsc#1184970": {
+        "description": ".*rpc\\.statd.*Failed to open directory sm.*|.*Failed to stat \/var\/lib\/nfs\/sm: No such file or directory",
+        "products": ["sle-micro", "microos"],
+        "type": "bug"
+    },
+    "bsc#1182500": {
+        "description": "Failed to transition into init label 'system_u:system_r:init_t:s0'",
+        "products": ["sle-micro", "microos"],
+        "type": "bug"
+    },
+    "bsc#1178033": {
+        "description": "kernel: ITS@0x8080000: Unable to locate ITS domain handle",
+        "products": ["sle-micro", "opensuse", "microos"],
+        "type": "bug"
+    },
+    "bsc#1177695": {
+        "description": "kernel: parport_pc: `none' invalid for parameter `dma'",
+        "products": ["sle-micro"],
+        "type": "bug"
+    },
+    "bsc#1177693": {
+        "description": "kernel: intel_powerclamp: CPU does not support MWAIT",
+        "products": ["sle-micro", "microos"],
+        "type": "bug"
+    },
+    "bsc#1182961": {
+        "description": "could not read from '/sys/module/pcc_pufreq/initstate': No such device",
+        "products": ["sle-micro", "microos"],
+        "type": "bug"
+    },
+    "bsc#1177028": {
+        "description": "Started bpfilter",
+        "products": ["sle", "opensuse"],
+        "type": "bug"
+    },
+    "bsc#1143125": {
+        "description": "systemd-vconsole-setup.service",
+        "products": ["sle", "opensuse"],
+        "type": "bug"
+    },
+    "apci-bridge": {
+        "description": "fail to add MMCONFIG information, can't access extended PCI configuration space under this bridge",
+        "products": ["sle", "opensuse"],
+        "type": "ignore"
+    },
+    "printk": {
+        "description": "kernel: printk: systemd:.*output lines suppressed due to ratelimiting",
+        "products": ["sle-micro", "microos", "sle", "opensuse"],
+        "type": "ignore"
+    },
+    "chrony": {
+        "description": "chronyd.*Received KoD RATE from",
+        "products": ["sle-micro", "microos", "sle", "opensuse"],
+        "type": "ignore"
+    }
```

Comment:
I have added bugrefs for_ vconsole_ and _bpfilter_ as of now and a introduced ignore type to skip expected messages as from _printk_, _chrony_ or kernel regarding _apci brigde_ ( not visible when I spawn a VM via virt-manager ).

How ever I am not really sure what to do with systemd timer related messages as http://kepler.suse.cz/tests/5625#step/journal_check/12. Any suggestions, please @jlausuch @Vogtinator ?

Reply:
Probably caused by `force_scheduled_tasks`, please try with #12752 applied.

Reply:
> Probably caused by `force_scheduled_tasks`, please try with #12752 applied.

Nice catch! Helps for the timers, however the related services report [Current command vanished from the unit file, execution of the command list won't be resumed](http://kepler.suse.cz/tests/5626#step/journal_check/12)

Reply:
Maybe I can shift the schedule of this module, it could possibly run before `force_scheduled_tasks`

Reply:
I shifted it around a bit more which might help, please try again. If it doesn't help, I'll report a systemd bug which we can use as reference.

> Maybe I can shift the schedule of this module, it could possibly run before force_scheduled_tasks

Why? `journal_check` did its job properly and found a bug!

Reply:
> I shifted it around a bit more which might help, please try again.

There's yet another case, which should be fixed - my VR no longer reports any of the errors now.

Reply:
Yup, neither mine after your PR. Let me merge yours and I will rebase this one. 
http://kepler.suse.cz/tests/5629#

Thanks for help!